src/go/k8s/apis/redpanda/v1alpha1/cluster_types.go

```diff
@@ -61,6 +53,31 @@ type StorageSpec struct {
 	StorageClassName string `json:"storageClassName,omitempty"`
 }
 
+// ExternalConnectivityConfig adds listener that can be reached outside
+// of a kubernetes cluster. The Service type NodePort will be used
+// to create unique ports on each Kubernetes nodes. Those nodes
+// need to be reachable from the client perspective. Setting up
+// any additional resources in cloud or premise is the responsibility
+// of the Redpanda operator user e.g. allow to reach the nodes by
+// creating new rule in AWS security group.
+// Inside the container the Configuration.KafkaAPI.Port + 1 will be
+// used as a external listener. This port is tight to the autogenerated
+// host port. The collision between Kafka external, Kafka internal,
+// Admin and RPC port is checked in the webhook.
+type ExternalConnectivityConfig struct {
+	// Enabled is the feature flag for registering additional listener
```

Comment:
nit: maybe just say that this enabled the feature? We already talked about that it changes listener config and other stuff

Reply:
Ok, changed.